pkg/util/util.go

```diff
@@ -22,3 +27,18 @@ func GetCertPool(paths []string) (*x509.CertPool, error) {
 	}
 	return pool, nil
 }
+
+func GetSecretValue(source *options.SecretSource) ([]byte, error) {
```

Comment:
Should this be a method on `SecretSource` directly in `options/common` above?

Reply:
Yeah could put it there, I wasn't really sure where to put it to be honest 😅  We could make it a method on the `SecretSource`, WDYT?